searx/templates/simple/preferences.html

```diff
@@ -76,7 +76,7 @@
           {{ icon('warning', 'The engine is not reliabled') }} {{ r -}}
       </span>{{- "" -}}
     </a>{{- "" -}}
-    <div class="engine-tooltip" style="right: 12rem;" role="tooltip" id="{{engine_name}}_reliability">
+    <div class="engine-tooltip" role="tooltip" id="{{engine_name}}_reliability">
```

Comment:
Without `right: 12rem` tooltip overflow outside of the screen:
![image](https://user-images.githubusercontent.com/1594191/132326460-1aac94ca-bfe3-4e27-afff-17bf0315eca0.png)

Reply:
done 717bf11